config/pure-python-without-pypy/gitignore

```diff
@@ -0,0 +1,14 @@
+*.egg-info
+*.pyc
+*.pyo
+.coverage
+.installed.cfg
+.mr.developer.cfg
+.tox/
+bin/
+coverage.xml
+develop-eggs/
+dist/
+htmlcov/
+lib/
+parts/
```

Comment:
What about `build/`?  And `eggs/`?

Reply:
I'm going to remove `build/` as I do not know where it is needed.
`eggs/` is the default place where eggs are stored by buildout if not using a global eggs directory.

Reply:
Sorry, not read correctly. I am going to add `eggs/` but what is `build/` for?

Reply:
build is created when using zestreleaser or such

Reply:
I think it's created when you build an egg in some situations, like a binary compile?

Reply:
It seems to be created when you run bdist_wheel

Reply:
most importantly: you can add build/ to gitignores ;) you'll never want it to be added or checked in

Reply:
Okay, I am going to add `build/`, too.